module/VuFind/src/VuFind/RecordTab/Factory.php

```diff
@@ -262,6 +262,13 @@ public static function getReviews(ServiceManager $sm)
     public static function getUserComments(ServiceManager $sm)
     {
         $capabilities = $sm->getServiceLocator()->get('VuFind\AccountCapabilities');
-        return new UserComments('enabled' === $capabilities->getCommentSetting());
+        $config = $sm->getServiceLocator()->get('VuFind\Config')->get('config');
+        $useRecaptcha = isset($config->Captcha) && isset($config->Captcha->forms)
+            && (trim($config->Captcha->forms) === '*'
+            || strpos($config->Captcha->forms, 'userComments'));
```

Comment:
In theory, you could refactor the controller plugin to expose the setting as a public method, and you could rewrite the factory to fetch the controller plugin out of the service manager and use it... but for four lines of code, it's probably just as easy to leave it alone for now!